ui/page/proposal/proposal_details_page.go

```diff
@@ -13,23 +13,25 @@ import (
 
 	"github.com/planetdecred/dcrlibwallet"
 	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/page/components"
 	"github.com/planetdecred/godcr/ui/renderers"
 	"github.com/planetdecred/godcr/ui/values"
+	"github.com/planetdecred/godcr/wallet"
 )
 
-const (
-	PageProposalDetails = "ProposalDetails"
-)
+const PageProposalDetails = "proposal_details"
 
 type proposalItemWidgets struct {
 	widgets    []layout.Widget
 	clickables map[string]*widget.Clickable
 }
 
 type proposalDetails struct {
-	theme              *decredmaterial.Theme
+	*load.Load
+	wallet *wallet.Wallet
```

Comment:
This struct is redundant and should not be used.

Reply:
Which of the struct is redundant? `proposalDetails` or `proposalItemWidgets`

Reply:
`wallet.Wallet` Is redundant.